Kite SDK/PSPrintSDK/OLCountry.h

```diff
@@ -18,4 +18,6 @@
 + (OLCountry *)countryForCode:(NSString *)code;
 + (OLCountry *)countryForCurrentLocale;
 
+- (BOOL)isEUCountry;
```

Comment:
Would you be adverse to me changing this to:

@property (nonatomic, readonly, getter=isInEurope) BOOL inEurope;

mainly because EU means something specific and Europe continent is more general